Update changelog.md [3.0.0] (#11)
MPDPilot/MPDPilot-website: _pages/changelog.md

---
# Change Log

## Release 3.0.0 `latest`
**Jan 25, 2024**

- Added support for multiple MPD server connections
- Automatic discovery of local MPD instances using Bonjour/Zeroconf protocol

> - Ability to define multiple local playback stream URLs (with various bitrate and audio quality)

Reviewer: ```suggestion
- Ability to define multiple local playback stream URLs (with various buffer sizes)
```

Author: Yes, but buffer size is only a tech detail of our configuration model. What is important for users is that they can define different MPD streaming outputs with various bitrate/quality. IMO better for marketing and more communicative for users outside of our detailed config context.

- Improved compatibility with Mopidy server

> - Local playback: fixed crashes when playing FLAC streams

Reviewer: ```suggestion
- Local playback: fixed crashes when playing FLAC streams
- Fixed navigation crash on iOS 17
```

- Numerous bug fixes and stability improvements

> 

Reviewer: ```suggestion
- Fixed navigation crash on iOS 17

```

Author: Isn't it contained within `Numerous bug fixes and stability improvements`?
